guidelines/sc/21/text-spacing.html

```diff
@@ -14,5 +14,8 @@ <h4>Text Spacing</h4>
    </ul>
  
  <p>Exception: Human languages and scripts that do not make use of one or more of these text style properties in written text can conform using only the properties that exist for that combination of language and script.</p>
+
+ <p class="note">Content is not required to use the values specified, the requirement is to ensure no-loss of content or functaionality when a user over-rides the authored styles with the values.</p>
```

Comment:
functaionality -> functionality

and, i think, over-rides -> overrides https://dictionary.cambridge.org/dictionary/english/override

Reply:
perhaps also: no-loss -> no loss, or 'ensure no-loss' -> avoid loss

Reply:
I'd personally go further than saying it's not required to use the value to say it's not implying or recommending these values be used as the default..  Nothing in the SC is encouraging their use as the default method at all and many folks would not find it useful - for me personally I would find content harder to read with those values.  The focus as we rightly say when they are applicable for the ability for them to be overwritten up to those values.

Reply:
I just thought it might help the objections if we were even stronger in the language that it wasn't just required but not recommended as the default.

Reply:
I'd rather avoid any 'recommend' language if we can, we have to focus on the requiments in the normative language / area.

Reply:
Sure, we could say something such as this does not imply these be used as the default values.